lib/interviewer/selectors/prop.js

```diff
@@ -22,7 +22,6 @@ export const getAdditionalAttributes = (stage, prompt) => {
 export const propStage = (_, props) => props?.stage ?? null;
 export const propPrompt = (_, props) => props?.prompt ?? null;
 export const propStageId = (_, props) => props?.stage?.id ?? null;
-export const getPropPromptId = (_, props) => props?.prompt?.id ?? null;
```

Comment:
I removed this entirely because (1) it is broken now that prompt isn't consistently passed to props, and (2) I wanted to force myself to find other instances where this broken selector was used and fix them.